### DISCUSSION TITLE
Plc4x Version 0.13.0 release date

### DISCUSSION THREAD

**Discussion by codekundan** [apache/plc4x#1949] [General]
Hi Plc4x Team, when can we expect Version 0.13.0 release.? Lately we have seen some bug fixes in 0.13.0-SNAPSHOT. Any info regarding  this.

**Comment by ottlukas** [comment]
Hi @codekundan,

We aim to release the new version build-tools:  https://github.com/apache/plc4x-build-tools until end of the year.

And @glcj seems to be working on finalizing the release for 0.13.0 by end of January 2025, if I understood his statement correctly.

**Comment by chrisdutz** [comment]
I just finalized the prerequisite for a new release (releasing the build-tools 1.8.0) ... however the "Project" on openCollective, that I setup hasn't yet received even a single cent. (https://opencollective.com/friends-of-toddy/projects/create-release-0130) I was planning on doing things this way, this time as the preparation of doing a release is actually quite a bit of work (Especially with our now 3 repositories) ... Updating the release scripts, updating the release notes, double checking everything, ... A lot of that I have already done, this campaign is more or less rewarding that work I put into that. I could probably promote the campaign explicitly, but besides the S7CommPlus driver campaign, nothing really gained any traction :-/

**Comment by glcj** [comment]
Hello,

I'm interested in finishing the stability tests for the S7 driver (it was doing some strange things when the machine came out of sleep, as reported by Chris), which I believe are now under control.

The Modbus issue is pretty well tested, so I think I should close the PR.

Best regards,
